stable/artifactory/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: artifactory
 home: https://www.jfrog.com/artifactory/
-version: 7.0.6
+version: 7.0.7
```

Comment:
Again, you're adding new features, you should bump the minor version.

Reply:
Ingress support was there. I fixed default backend